### DISCUSSION TITLE
Filtering Vulnerability Data Using host_info.groups

### DISCUSSION THREAD

**Discussion by kpoznanski5** [CrowdStrike/psfalcon#290] [Q&A] [answered]
Hello,

My name is Kris. I am using the PSFalcon to export the vulnerability data. First I ran an example script which had the following filter:
Filter = "status:!'closed'+(created_timestamp:>'now-1d'+created_timestamp:<'now')+suppression_info.is_suppressed:false"
This one works just fine.
However, I can't get the filter working using the host_info.groups property.  I have tried the following:
Filter = "status:!'closed'+host_info.groups:'Group1'+suppression_info.is_suppressed:false"
or
Filter = "status:!'closed'+host_info.groups.name:'Group1'+suppression_info.is_suppressed:false"

But both return [{"code":400,"message":"invalid input data"}]

I am using the latest version of the PSFalcon.

Any help on resolving this issue would be appreciated.

Thank you.

Kris

**Comment by bk-cs** [comment, marked as answer]
The `host_info.groups` filter expects a group identifier and does not accept a name. See the Spotlight API documentation for examples:

\[ [EU-1](https://falcon.eu-1.crowdstrike.com/documentation/98/spotlight-apis#filter-spotlight-apis) | [US-1](https://falcon.crowdstrike.com/documentation/98/spotlight-apis#filter-spotlight-apis) | [US-2](https://falcon.us-2.crowdstrike.com/documentation/98/spotlight-apis#filter-spotlight-apis) | [US-GOV-1](https://falcon.laggar.gcw.crowdstrike.com/documentation/98/spotlight-apis#filter-spotlight-apis) \]

Give this a try:
```powershell
Get-FalconVulnerability -Filter "status:!'closed'+host_info.groups:['44df8f70a7cc4eb495a9977c07594c95']+suppression_info.is_suppressed:false"
```

**Reply by kpoznanski5** [reply]
Thank you. We were able to get the groups IDs and now the script works with group filterting.
Thank you.

Kris
